clientv2/vpp/dbadapter/data_resync_db.go

```diff
@@ -65,6 +66,15 @@ func (dsl *DataResyncDSL) ACL(val *acl.ACL) vppclient.DataResyncDSL {
 	return dsl
 }
 
+// ACL adds Access Control List to the RESYNC request.
+func (dsl *DataResyncDSL) ABF(val *abf.ABF) vppclient.DataResyncDSL {
+	key := abf.Key(val.Index)
```

Comment:
Using `models.Key` for the `val` variable actually prevents extra allocation and should be preferred. The `xxx.Key` are for cases when you don't actually have entire model data, e.g. having name of interface for dependency.